rac-gcp-deploy/orb.yml

```diff
@@ -288,6 +281,115 @@ jobs:
           command: sbt service-test:test
           no_output_timeout: 20m
 
+  snyk_test:
+    parameters:
+      executor:
+        description: "Name of executor to use for this job. Defaults to machine executor"
+        type: executor
+        default: docker
+      workspace-dir:
+        description: "Path to restore the workspace"
+        type: string
+        default: ~/project
+      cache-suffix:
+        description: "Suffix to use for the cache key, e.g. -v2, to be able to clear the cache"
+        default: ""
+        type: string
+    executor: <<parameters.executor>>
+    steps:
+      - attach_workspace:
+          at: <<parameters.workspace-dir>>
+      - load_cache:
+          cache-suffix: <<parameters.cache-suffix>>
+      - run: *sbt-dependency-graph
+      - snyk/scan:
+          project: $CIRCLE_PROJECT_REPONAME
+          fail-on-issues: true
+          monitor-on-build: false
+          severity-threshold: high
+
+  snyk_monitor:
+    parameters:
+      executor:
+        description: "Name of executor to use for this job. Defaults to machine executor"
+        type: executor
+        default: docker
+      workspace-dir:
+        description: "Path to restore the workspace"
+        type: string
+        default: ~/project
+      cache-suffix:
+        description: "Suffix to use for the cache key, e.g. -v2, to be able to clear the cache"
+        default: ""
+        type: string
+    executor: <<parameters.executor>>
+    steps:
+      - attach_workspace:
+          at: <<parameters.workspace-dir>>
+      - load_cache:
+          cache-suffix: <<parameters.cache-suffix>>
+      - run: *sbt-dependency-graph
+      - snyk/scan:
+          project: $CIRCLE_PROJECT_REPONAME
+          fail-on-issues: false
+          monitor-on-build: true
+
+  pr_comment_snyk:
```

Comment:
IMHO these should be commands, not jobs. Jobs require start of the new executor which increases contention on CircleCI and makes builds longer

Reply:
By these, you are referring to all of the snyk jobs or just the `pr_comment_snyk`one?

Reply:
I can make the snyk_monitor a job and include it as a step of the release job.
As for the other two jobs, I don't think they be made simple commands and included as a step on the existing jobs. These should be run only for PR's but we don't any specific jobs only for PRs.
I happy to hear any suggestions you might have.